activestorage/app/models/active_storage/blob.rb

```diff
@@ -143,6 +143,16 @@ def variable?
     ActiveStorage.variable_content_types.include?(content_type)
   end
 
+  # Returns the default content disposition (+:inline+ or +:attachment+) to be used when serving the blob. +:inline+ is used
+  # in all cases except when its content type should be rendered as binary, that is, when it's included in
+  # +ActiveStorage.content_types_to_render_as_binary+.
+  def content_disposition
+    if ActiveStorage.content_types_to_render_as_binary.include?(content_type)
+      :attachment
+    else
+      :inline
+    end
+  end
```

Comment:
I had put this method as `private` before, then I thought it could make sense to expose it, for example if we were to check this for blob's variants in the future (although none of the variable content types makes sense as attachment right now).

Reply:
I think we should keep it private for now. If we need to call it elsewhere, we can make it public at that time.

Let’s also rename it to `default_disposition` to clarify that it’s a fallback.